tests/io/test.crx.js

```diff
@@ -16,16 +16,16 @@ const chromeManifestEntry = Object.assign({}, defaultData, {
   fileName: 'chrome.manifest',
 });
 
-const installRdfEntry = Object.assign({}, defaultData, {
+const installFileEntry = Object.assign({}, defaultData, {
   compressedSize: 416,
   uncompressedSize: 851,
-  fileName: 'install.rdf',
+  fileName: 'file.entry',
 });
 
-const dupeInstallRdfEntry = Object.assign({}, defaultData, {
+const dupeInstallFileEntry = Object.assign({}, defaultData, {
   compressedSize: 416,
   uncompressedSize: 851,
-  fileName: 'install.rdf',
+  fileName: 'file.entry',
 });
```

Comment:
I think it's more real-life if we call these `manifest.json` instead since that's the correct replacement for `install.rdf` in the WebExtension world.